tests/ui/unnecessary_filter_map.rs

```diff
@@ -1,3 +1,4 @@
+//@no-rustfix
```

Comment:
Is the whole test rustfix-incompatible? If not, could we split our a `-fix` test file?